java/ql/lib/ext/javax.script.model.yml

```diff
@@ -3,4 +3,5 @@ extensions:
       pack: codeql/java-all
       extensible: sinkModel
     data:
+# suggested label is not supported:      - ["javax.script", "Compilable", True, "compile", "(String)", "", "Argument[0]", "command-injection", "ai-manual"]
```

Comment:
This is rather a script injection sink, and it's handled by more specific queries (like MVEL injection or XSLT injection). We even have a general experimental query that uses this sink, but it doesn't seem we should add it to the command injection query.

```suggestion
```